### DISCUSSION TITLE
guestbook

### DISCUSSION THREAD

**Discussion by giscus** [kuohuanhuan/blog#3] [Comments]
# guestbook

嗨 👋，感謝你到訪我的網站 😀——這裡是留言板，歡迎留言 😉

https://nekohuan.cyou/guestbook

**Comment by timcheng5514** [comment]
早安
### 某粽子到此一遊
~~好像是來亂的~~但居然是第一個（

**Comment by TokdEd** [comment]
開發者您好：
沒事

**Comment by orenccl** [comment]
Hi

**Reply by cnrkuo** [reply]
哈囉！

**Comment by kuang2714** [comment]
博主你好，茶备案项目新开张，希望得到更大的支持加入茶备案！注册得到您的专属网站备案号！希望博主多多支持！https://icp.redcha.cn
希望博主能在我们网站申请一个备案号放在页脚呀！感谢博主的支持
无论赞助多少，或者是不赞助也好联系站长都可以，尽量给个靓号
如果被打扰了，那就万分抱歉
